sycl/include/CL/__spirv/spirv_ops.hpp

```diff
@@ -51,33 +51,30 @@ template <typename T, typename Tp, std::size_t R, std::size_t C,
           __spv::Scope::Flag S = __spv::Scope::Flag::Subgroup>
 extern __DPCPP_SYCL_EXTERNAL
     __spv::__spirv_JointMatrixINTEL<Tp, R, C, L, S, U> *
-    __spirv_CooperativeMatrixConstructCheckedINTEL(const T Value, size_t Height,
-                                                   size_t Stride, size_t Width,
-                                                   size_t CoordX,
-                                                   size_t CoordY);
+    __spirv_CooperativeMatrixConstructCheckedINTEL(size_t CoordX, size_t CoordY,
+                                                   size_t Height, size_t Width,
+                                                   const T Value);
 
 template <typename T, typename Tp, std::size_t R, std::size_t C,
           __spv::MatrixUse U,
           __spv::MatrixLayout L = __spv::MatrixLayout::RowMajor,
           __spv::Scope::Flag S = __spv::Scope::Flag::Subgroup>
 extern __DPCPP_SYCL_EXTERNAL
     __spv::__spirv_JointMatrixINTEL<Tp, R, C, L, S, U> *
-    __spirv_JointMatrixLoadCheckedINTEL(T *Ptr, std::size_t Stride,
-                                        size_t Height, size_t Width,
-                                        size_t CoordX, size_t CoordY,
-                                        __spv::MatrixLayout Layout = L,
-                                        __spv::Scope::Flag Sc = S,
-                                        int MemOperand = 0);
+    __spirv_CooperativeMatrixLoadCheckedINTEL(
+        T *Ptr, size_t CoordX, size_t CoordY, __spv::MatrixLayout Layout = L,
+        size_t Height = 0, size_t Width = 0, std::size_t Stride = 0,
```

Comment:
do we need to set default values for these variables? It is not part of the spec, is it?

Reply:
Spec doesn't have them indeed. But SYCL headers implementation sets default value for layout, so following C++ rules any other value must be defaulted.